edweiss-app/components/memento/CardListDisplayComponent.tsx

```diff
@@ -77,24 +78,24 @@ export const CardListDisplay: React.FC<{
             m='md' mt={'sm'} mb={'sm'} p='lg'
             backgroundColor={isSelected ? 'rosewater' : 'base'}
             borderColor='crust' radius='lg'
+            b={'xl'}
         >
             <TView flexDirection='row' justifyContent='space-between'>
-                <TView flex={1} mr='md'>
-                    <TText bold color='text' ellipsizeMode='tail' numberOfLines={1}>
+                <TView testID={`cardQuestionIndex_${cardIndex}`} flex={1} mr='md'>
+                    {/*<TText bold color='text' ellipsizeMode='tail' numberOfLines={1}>
+                        {card.question}
```

Comment:
Can we remove this commented code?

Reply:
I'd prefer to keep the commented code because for now because Rich Text is still in development so in the worst case, I will re-use the commented code here.